sdk/azcore/policy/policy_test.go

```diff
@@ -52,3 +53,13 @@ func TestWithRetryOptions(t *testing.T) {
 		t.Fatalf("unexpected value %d", opts.MaxRetries)
 	}
 }
+
+func TestIncludeResponse(t *testing.T) {
+	ctx := IncludeResponse(context.Background())
+	require.NotNil(t, ctx)
+	raw := ctx.Value(shared.CtxIncludeResponseKey{})
+	_, ok := raw.(**http.Response)
+	if !ok {
+		t.Fatalf("unexpected type %T", raw)
+	}
```

Comment:
I believe you can use require here too:
```suggestion
        require.Truef(t, ok, "unexpected type %T", raw)
```

Reply:
Good idea, fixed.

Reply:
I was hoping that there was a "required type" check in testify but I couldn't find it.

Reply:
OK there is `require.IsType` but that's a little awkward for this case.